### PR TITLE
User/bsmithvt/access control script

### PR DESCRIPTION
**Pull request by bsmithvt**

### CODE REVIEW

File: Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1 [hunk 1]
```diff
@@ -758,4 +757,25 @@ Write-Host "There are $($edwAdminUsers.Count) users with this role"
...
 
+#Register Fabric.Authorization.AccessControl client
+$body = @'
+{
+    "clientId":"fabric-access-control", 
+    "clientName":"Fabric Authorization Access Control Client", 
+    "requireConsent":"false", 
+    "allowedGrantTypes": ["implicit"], 
+    "allowedScopes": ["openid", "profile", "fabric.profile", "fabric/authorization.read", "fabric/authorization.write", "fabric/idprovider.searchusers", "fabric/authorization.dos.write"],
+	"allowedCorsOrigins": ["$authorizationServiceUrl/v1"],
```
Comment (michaelvidal24) [inline Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1]: The allowed cors origins should just be the URL of the root web server.

File: Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1 [hunk 2]
```diff
@@ -758,4 +757,25 @@ Write-Host "There are $($edwAdminUsers.Count) users with this role"
...
+	"redirectUris": ["$authorizationServiceUrl/v1/oidc-callback.html", "$authorizationServiceUrl/v1/silent.html"],
```
Comment (michaelvidal24) [inline Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1]: Do we need the v1's here? I don't think users should need to put in a /v1 to get to the UI, they should just be able to enter something like https://host.domain.local/Authorization. The UI should know if its using a v1 or v2 endpoints of the API.

File: Fabric.Authorization.API/Modules/RootModule.cs [hunk 3]
```diff
@@ -11,7 +12,7 @@ public RootModule(): base("/v1")
 
         private dynamic Redirect()
         {
-            return Response.AsRedirect("/client/index.html");
+            return Response.AsRedirect($"/{AccessControl.Path}/index.html");
```
Comment (michaelvidal24) [inline Fabric.Authorization.API/Modules/RootModule.cs]: Will this allow us to hit the app using https://host.domain.local/Authorization, or do we have to enter:
https://host.domain.local/Authorization/accesscontrol?

Comment (bsmithvt) [reply Fabric.Authorization.API/Modules/RootModule.cs]: Yes, when I run locally and go to `localhost:5004`, I get redirected to the landing page for the access control UI.

**Pull request merged**